--- spellChecker.js	2009-12-16 11:41:57.000000000 +0100
+++ spellChecker.js	2009-12-16 11:27:58.000000000 +0100
@@ -13,13 +13,13 @@
 function spellChecker( textObject ) {
 
 	// public properties - configurable
-	this.popUpUrl = '/speller/spellchecker.html';							// by FredCK
+//	this.popUpUrl = '/speller/spellchecker.html';							// by FredCK
 //	this.popUpUrl = 'fck_spellerpages/spellerpages/spellchecker.html';		// by FredCK
 	this.popUpName = 'spellchecker';
-	this.popUpProps = "menu=no,width=440,height=350,top=70,left=120,resizable=yes,status=yes";	// by FredCK
-//	this.popUpProps = null ;																	// by FredCK
-	this.spellCheckScript = '/speller/server-scripts/spellchecker.php';		// by FredCK
-	//this.spellCheckScript = '/cgi-bin/spellchecker.pl';
+//	this.popUpProps = "menu=no,width=440,height=350,top=70,left=120,resizable=yes,status=yes";	// by FredCK
+	this.popUpProps = null ;																	// by FredCK
+//	this.spellCheckScript = '/speller/server-scripts/spellchecker.php';		// by FredCK
+//	this.spellCheckScript = '/cgi-bin/spellchecker.pl';
 
 	// values used to keep track of what happened to a word
 	this.replWordFlag = "R";	// single replace
