--- /ws/Playground/ASpell/dialogs/aspell.js	2009-12-16 11:39:14.000000000 +0100
+++ dialogs/aspell.js	2009-12-14 13:22:34.000000000 +0100
@@ -32,7 +32,8 @@
 				
 				// Create spellcheck object, set options/attributes
 				var oSpeller = new spellChecker(document.getElementById(textareaId));
-				oSpeller.spellCheckScript = editor.plugins.aspell.path+'spellerpages/server-scripts/spellchecker.php';
+				oSpeller.spellCheckScript = CKEDITOR.config.spellerPagesServerScript;
+				//oSpeller.spellCheckScript = editor.plugins.aspell.path+'spellerpages/server-scripts/spellchecker.php';
 				oSpeller.OnFinished = function (numChanges) { oSpeller_OnFinished(dialog, numChanges) };
 				oSpeller.popUpUrl = editor.plugins.aspell.path+'spellerpages/spellchecker.html';
 				oSpeller.popUpName = iframeId;
